prestoadmin/server.py

```diff
@@ -149,17 +149,26 @@ def wait_for_presto_user():
 def uninstall():
     """
     Uninstall Presto after stopping the services on all nodes
+
+    Args:
+        --nodeps (optional): Flag to indicate if server uninstall
+        should ignore checking Presto rpm package dependencies.
+        Equivalent to adding --nodeps flag to rpm -e
     """
     stop()
 
+    nodeps = ''
+    if env.nodeps:
+        nodeps = ' --nodeps'
+
     # currently we have two rpm names out so we need this retry
     with quiet():
-        ret = sudo('rpm -e presto')
+        ret = sudo('rpm -e%s presto' % (nodeps))
```

Comment:
I would rather include the the space as part of the formatting string instead of as part of variable that gets substituted in.